llvm/lib/CodeGen/AssignmentTrackingAnalysis.cpp

```diff
@@ -1052,9 +1052,9 @@ class AssignmentTrackingLowering {
       if (Source.isNull())
         OS << "null";
       else if (isa<DbgAssignIntrinsic *>(Source))
```

Comment:
shouldn't these use dyn_cast? It is isa + cast.

Reply:
That's much cleaner.  Thanks!  Updated in the latest revision.